OmniBLE/OmnipodCommon/UnfinalizedDose.swift

```diff
@@ -228,6 +230,12 @@ public struct UnfinalizedDose: RawRepresentable, Equatable, CustomStringConverti
         } else {
             self.automatic = false
         }
+
+        if let isLowTemp = rawValue["isLowTemp"] as? Bool {
+            self.isHighTemp = isLowTemp
```

Comment:
If this odd looking code is correct, a comment would be helpful.

Reply:
Ah, thanks; refactoring error.